docs/3.0.0-beta.x/deployment/heroku.md

```diff
@@ -58,7 +58,7 @@ Follow the instructions and return to your command line.
 
 Create a [new Strapi project](../getting-started/quick-start.md) (if you want to deploy an existing project go to step 4).
 
-::: warning NOTE
```

Comment:
Hello `tip` display the message in green
and `warning` display the message in yellow.

Can you please keep `warning` when it's used.
This id for everywhere you changed `warning` by `tip`.

Thank you.

Reply:
Thanks for your review.